src/Effect/App.js

```diff
@@ -0,0 +1,66 @@
+import { applyMiddleware, createStore } from 'redux'
+import { combineEpics, createEpicMiddleware } from 'redux-observable'
+import { compose, flip, uncurryN } from 'ramda'
+import { render } from 'react-dom'
+import { tryCatch, chain } from 'crocks'
+import App from '../Component/App'
+import React from 'react'
+import rootReducer from '../State'
+
+
+// rootEpic :: (Observable Action, Observable State) -> Observable Action
+export const rootEpic = combineEpics()
+
+
+// configureStore :: () -> Result Error Redux.Store
+export const configureStore = tryCatch(() => {
+  const env = process.env.NODE_ENV || 'development'
+  const debug = Boolean(process.env.REACT_APP_DEBUG_STATE || 0)
+  const reducer = debug
+    ? debugReducerDecorator(rootReducer)
+    : rootReducer
+
+  if (env === 'test') {
+    return createStore(rootReducer)
+  }
+
+  const epicMiddleware = createEpicMiddleware()
```

Comment:
Could be nice to describe dependencies injection here ( https://redux-observable.js.org/docs/api/createEpicMiddleware.html ).
All the utilities used by the epics (such as fetch) should be injected as deps here as it will ease the tests of epics ;)

Reply:
Yep, why not. We need to write documentation (probably in a wiki). We can then add a section about thoses dependencies